src/components/views/login/VectorLoginHeader.js

```diff
@@ -23,11 +23,14 @@ module.exports = React.createClass({
     statics: {
         replaces: 'LoginHeader',
     },
+    propTypes: {
+        icon: React.PropTypes.string,
+    },
 
     render: function() {
         return (
             <div className="mx_Login_logo">
-                <img src="img/logo.png" width="195" height="195" alt="Riot"/>
+                <img src={this.props.icon || "img/logo.png"} width="195" height="195" alt="Riot"/>
```

Comment:
Will icons always be square? May want to only set one of these dimensions and give it a bounding box if necessary to stop things around it jumping about after the image loads.